lib/chewy/index.rb

```diff
@@ -30,22 +35,31 @@ class Index
     #
     def self.index_name(suggest = nil)
       if suggest
-        @index_name = build_index_name(suggest, prefix: default_prefix)
+        @index_name = build_index_name(suggest, prefix: config[:prefix])
       else
         @index_name ||= begin
           build_index_name(
             name.sub(/Index\Z/, '').demodulize.underscore,
-            prefix: default_prefix
+            prefix: config[:prefix]
           ) if name
         end
       end
       @index_name or raise UndefinedIndex
     end
 
-    # Prefix to use
+    def self.use_client(name)
+      #puts "use_client: #{name}"
+      self.client_name = name.to_sym
+    end
+
+    # Elasticsearch::Client to use
     #
-    def self.default_prefix
```

Comment:
Why did you get rid of this method? It was explicitly added in order to be able to change the prefix from every index. It was like a feature :)